editor.planx.uk/src/@planx/components/FileUpload/Public.tsx

```diff
@@ -34,7 +34,7 @@ const slotsSchema = array()
   .required()
   .test({
     name: "nonUploading",
-    message: "Upload at least one file.",
+    message: "Upload at least one file",
```

Comment:
minor un-related tidy up here: our error messages typically don't use punctuation, so just updating this one too for consistency !